src/data/java_profile.rs

```diff
@@ -211,6 +212,19 @@ impl CollectData for JavaProfileRaw {
         }
 
         let data_dir = PathBuf::from(params.data_dir.clone());
+        for key in self.process_map.keys() {
+            let mut html_path = data_dir.clone();
+            html_path.push(format!("{}-java-flamegraph-{}.html", params.run_name, key));
+
+            let html_loc = html_path.to_str().unwrap();
+            let tmp_loc = format!(
+                "/tmp/aperf_tmp/{}-java-flamegraph-{}.html",
+                params.run_name, key
+            );
+
+            fs::copy(tmp_loc.clone(), html_loc).ok();
```

Comment:
A move would be better than a copy. Although, the answer to my question of why we have to write to /tmp/ in the first place might mean it has to be a copy.